WallTrailingGame/src/main/res/values/strings.xml

```diff
@@ -9,10 +9,9 @@
     <string name="failed_permissions">Permissions required!</string>
 
     <string name="receivers_text">Receivers for Keys: </string>
-    <string-array name="receivers">
-        <item>"none"</item>
-    </string-array>
+
     <string-array name="distance">
+        <item>Wall Distance (feet)</item>
```

Comment:
Make `Wall Distance (feet)` a constant string in the strings.xml, so we can reference it as a variable.